featuretools/tests/entityset_tests/test_es.py

```diff
@@ -131,16 +131,17 @@ def test_add_relationship_instantiated_logical_types(es):
                      time_index='datetime')
     assert 'log2' in es.dataframe_dict
     assert es['log2'].ww.schema is not None
-    assert es['log2'].ww.logical_types['product_id'] == ltypes.Categorical()
-    assert es['products'].ww.logical_types['id'] == ltypes.Categorical
+    # --> see if we need to change this test to test something else now... not sure this will still fail :/
+    assert isinstance(es['log2'].ww.logical_types['product_id'], ltypes.Categorical)
+    assert isinstance(es['products'].ww.logical_types['id'], ltypes.Categorical)
 
     warning_text = 'Logical type Categorical for child column product_id does not match parent '\
         'column id logical type Categorical. There is a conflict between the parameters. '\
         'Changing child logical type to match parent.'
     with pytest.warns(UserWarning, match=warning_text):
         es.add_relationship(u'products', 'id', 'log2', 'product_id')
```

Comment:
This warning no longer gets thrown, because initializing with `Categorical` or `Categorical()` now results in the same logical type. The way to trigger this warning now will be to have two logical types that are the same but have different parameters. For example `Datetime()` and `Datetime(datetime_format)`. However, in this case, the columns need to be parent and child columns in a relationship, and none of the index columns in the mock ecommerce entityset have parameters, so I'm not totally sure how we're going to cover this line. 

One option might be to change the logical types to be `Ordinal`, but the parameters would need to be different, and the `order` specified in Ordinal probably won't be different between parent and child ids. 

The other option is to create a new fixture for this test.

Reply:
New fixture / modifying the fixture when it gets passed to the test so that there are different parameters sounds good

Reply:
figured out how to do it without having to create a new fixture! (I forgot that we can add elements to `order` that aren't in the series, so that way we can have different orders between child and parent columns!

Reply:
figured out how to do it without having to create a new fixture! (I forgot that we can add elements to `order` that aren't in the series, so that way we can have different orders between child and parent columns!